distribution/lib/Standard/Database/0.0.0-dev/src/Data/Internal/Base_Generator.enso

```diff
@@ -50,7 +51,7 @@ make_binary_op name =
                 op = Sql.code " "+name+" "
                 (arguments.at 0)++op++(arguments.at 1) . paren
             False ->
-                Error.throw ("Invalid amount of arguments for operation " + name)
+                Error.throw <| Illegal_State_Error ("Invalid amount of arguments for operation " + name)
```

Comment:
There is an Arity_Error we could use for this

Reply:
Hmm, good point but I think it is specifically for polyglot calls, no?

I'm not sure if re-using it in such a different context is a good idea. Maybe create a sibling one for SQL? Anyway - these should never happen in practice - they'd indicate a bug in the IR generator so it is more of an internal error anyway. The user facing ones are `Unsupported_Database_Operation_Error`.

Reply:
Happy to have `Unsupported_Database_Operation_Error`, but also no reason we can't reuse `Arity_Error` as we need (though I grant you a complicated name for a simple concept!)